codec/packer_test.go

```diff
@@ -125,13 +125,13 @@ func TestNewReader(t *testing.T) {
 	vInt := 900
 	wp := NewWriter(5, 5)
 	// Add an int and a bool
-	wp.PackInt(vInt)
+	wp.PackInt(int32(vInt))
```

Comment:
nit: declare `vInt` as the desired type above instead of casting twice here and below